### DISCUSSION TITLE
Convert Model Cards to / from Huggingface Hub model cards

### DISCUSSION THREAD

**Discussion by hanneshapke** [tensorflow/model-card-toolkit#234] [Integrations]
What do you all think about a converter tool to convert model cards to and from Huggingface Hub model cards?

More info about HF model cards: https://huggingface.co/docs/hub/model-cards

**Comment by codesue** [comment]
Let's do it!

This is a larger project, but we can approach it incrementally, beginning with rendering a mct.ModelCard to a markdown template that fits the Hugging Face schema.

Here are some other helpful resources:
- [schema](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1)
- [template](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md)
- [annotation guide](https://huggingface.co/docs/hub/model-card-annotated)
- [annotation tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)
- [HF ModelCard API ](https://huggingface.co/docs/huggingface_hub/guides/model-cards)
